### PR TITLE
Updated title and author list

### PR DESCRIPTION
**Pull request by aashish24**
@alvarosan @sankhesh please review. As for the title goes, I am open for suggestions and I will highly appreciate any feedback.

### CODE REVIEW

File: title.tex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-\title{Practical Volume Rendering using Programmable Shaders in VTK for Scientific and Medical Computing}
+\title{High Performance Ubiquitous Volume Rendering using Programmable Shaders in VTK for Scientific and Medical Computing}
```
Comment (sankhesh) [inline title.tex]: How about changing the word *Computing* to *Visualization* ?

Comment (alvarosan) [reply title.tex]: +1

Comment (sankhesh) [inline title.tex]: @aashish24 I know I said leaving high performance but on second thought I'd say 

"Ubiquitous volume rendering using programmable shaders in VTK for scientific and medical [visualization]" has a nice ring to it - thoughts?

Minor - no need to capitalize intermediate words in the title.

**Comment by alvarosan** [comment]
Also consider _Universal_ instead of _Ubiquitous_ , although I would remove it altogether and leave it as _High Performance Volume Rendering ..._ .    My 2 cents.

File: title.tex (hunk 1 above)
Comment (aashish24) [reply title.tex]: I am fine with Visualization instead of Computing. @alvarosan thoughts?

**Comment by sankhesh** [comment]
Changed title based on discussion with @aashish24 and @alvarosan in 61c4c8c

**Pull request merged**